packages/pds/src/db/database-schema.ts

```diff
@@ -54,4 +55,6 @@ export type DatabaseSchema = user.PartialDB &
   sceneMemberCount.PartialDB &
   sceneVotesOnPost.PartialDB
 
+export type DatabaseSchema = Kysely<DatabaseSchemaType>
```

Comment:
We use this enough that I just decided to export it as the `Kysely` wrapped type.

I had to introduce (the somewhat sloppily named 😛 ) `DatabaseSchemaType` for a few cases that the underlying type